WeakAuras/Features.lua

```diff
@@ -29,11 +32,16 @@ function Features:Enabled(id)
 end
 
 ---@param id string
-function Features:Enable(id)
+---@param auto? boolean
+function Features:Enable(id, auto)
   if self:Exists(id) and not self.__feats[id].enabled then
     self.__feats[id].enabled = true
-    if self.__feats[id].persist then
-      Private.db.features[id] = true
+    if self.__feats[id].persist and not auto then
+      if self.hydrated then
+        Private.db.features[id] = true
+      else
```

Comment:
How can the code in the else branch run? 

Enable( id, false) seems to come from Hydrate/user cli usage, both of which can imho only run after hydrated is set to true.

Reply:
`auto` is not a required parameter, so `Features:Enable(id)` (from e.g. user action) will take this branch because `auro` is nil.

Reply:
Ah, I read your comment a second time and it makes sense now. I think I was worried about an edge case that I never ended up creating.